src/platform/Ameba/ConnectivityManagerImpl.cpp

```diff
@@ -787,5 +798,66 @@ void ConnectivityManagerImpl::DHCPProcess(void)
     xTaskCreate(DHCPProcessThread, "DHCPProcess", 4096 / sizeof(StackType_t), this, 1, NULL);
 }
 
+CHIP_ERROR ConnectivityManagerImpl::_GetWiFiSecurityType(uint8_t & securityType)
```

Comment:
This and following ... do not seem like correct implementations at all.  We should not be returning "no error" if we can't get valid values!

Reply:
Hi @bzbarsky-apple ,
Many thanks for the review, for this implementation, I was actually referring to your pervious comment on
https://github.com/project-chip/connectedhomeip/pull/10313/files/675c1ebb4b1af920aecc3208b891c048ce0382d9#r729519667
Therefore, I'd like to know if there is already a state to represent null function we should use?

Reply:
On the wire we can represent null now.

We should fix this API to take a `DataModel::Nullable<uint8_t> &` and then go from there.

That said, I guess we can do it as a followup to this PR, since this is no worse than other platforms...